### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by laurentpayot** [dillonkearns/elm-typescript-interop#36]
I was considering using `elm-typescript-interop` but I ran into errors like this:
```
Could not parse file `src/elm/Page/SignIn.elm` at position 11961. Errors:
expected end of input
```
Looking for similar issues I found [this comment](https://github.com/dillonkearns/elm-typescript-starter/issues/8#issuecomment-608156777) about this project *maybe* not being maintained any more.

This project is such a great idea as TypeScript is becoming a standard for front end dev. Is it dead yet ?

**Comment by dillonkearns** [comment]
Someone reached out to me a while ago with a rewrite of this project in Elm 0.19 with the latest `elm/parser`.

I don't plan to upgrade this specific repo to Elm 0.19 under the hood, or changing it over to `stil4m/elm-syntax`. This other project attempts to rewrite it using that new set of tooling, and if it can handle the same cases that this library can then I'll call it an official replacement. I'm not sure where that project stands, it may be on hold.

**Comment by dillonkearns** [comment]
I've published [`elm-ts-interop`](https://www.npmjs.com/package/elm-ts-interop) and it's now a stable project. This project (`elm-typescript-interop`) is now officially deprecated in favor of the new `elm-ts-interop` NPM package. The new package gives more flexibility to send things like custom types, and it also uses a single port pair (to and from) which has some cool features like enabling you to do an exhaustive switch in JS to make sure you handle all messages from Elm.

I wrote about the reasons I used a new approach in this post [Types Without Borders Isn't Enough](https://incrementalelm.com/types-without-borders-isnt-enough/). We also did [an Elm Radio podcast episode](https://elm-radio.com/episode/elm-ts-interop), and [there are some additional resources on the docs site](https://elm-ts-interop.com/resources). I hope y'all enjoy the new library!
